src/Fields/BelongsTo.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Larsklopstra\Nebula\Fields;
+
+use Larsklopstra\Nebula\Contracts\NebulaField;
+use Larsklopstra\Nebula\Fields\Concerns\HasHelperText;
+use Larsklopstra\Nebula\Fields\Concerns\HasResource;
+
+class BelongsTo extends NebulaField
+{
+    use HasHelperText, HasResource;
```

Comment:
I feel like all `NebulaField` instances should have access to the resource, so perhaps it makes more sense to add the trait to the parent class instead.

Reply:
Maybe a naming issue, but this is a `BelongsTo` resource, so a `PostResource` is able to resolve the `UserResource`, explained it earlier a little^

Reply:
Yeah, perhaps it's a naming issue then.